java/system-test/system-test-drivers/src/main/java/sleeper/systemtest/drivers/cdk/TearDownMavenSystemTest.java

```diff
@@ -74,11 +78,16 @@ public static void tearDown(
                 cloudFormation.deleteStack(builder -> builder.stackName(instanceId));
             } catch (RuntimeException e) {
                 LOGGER.warn("Failed deleting instance stack: " + instanceId, e);
+                tearDownFailures.add(new TearDownFailure(instanceId, e));
```

Comment:
If a stack already been deleted, I think this might fail the whole script, whereas previously this would just log the exception. The normal use case of this script is a bulk tear down over many instances and test executions. In that case, many of the combined instance IDs may never have really been deployed.

I think we'd like the script to still pass if the stack has already been deleted.

Reply:
It looks like even if the stack does not exist an exception isnt thrown here. I've just tested in AWS with some non-existent instances and the exit code is still 0. The delete stack call just silently fails, and the WaitForStackToDelete passes since the stack does not exist.

I'm not sure if an exception can realistically be thrown here, maybe if there is a network issue?

Reply:
I guess we shouldn't catch it at all then, thanks for checking!